Makefile.node

```diff
@@ -96,15 +102,19 @@ test-unit:
 
 test-unit-coverage:
 	@if [ -d test/unit ]; then \
-		if [ -x $(NPM_BIN)/istanbul ]; then \
-			istanbul cover $(NPM_BIN)/_mocha -- test/unit --recursive && $(TASK_DONE); \
+		if [ -x $(NPM_BIN)/nyc ]; then \
+			nyc --reporter=text --reporter=html $(NPM_BIN)/_mocha test/unit --recursive && $(TASK_DONE); \
 		else \
-			make test-unit; \
+			if [ -x $(NPM_BIN)/istanbul ]; then \
+				istanbul cover $(NPM_BIN)/_mocha -- test/unit --recursive && $(TASK_DONE); \
+			else \
+				make test-unit; \
+			fi \
 		fi \
 	fi
 
 test-integration:
-	@if [ -d test/integration ]; then mocha test/integration --timeout $(INTEGRATION_TIMEOUT) --slow $(INTEGRATION_SLOW) && $(TASK_DONE); fi
+	@if [ -d test/integration ]; then mocha test/integration --timeout $(INTEGRATION_TIMEOUT) --slow $(INTEGRATION_SLOW) $(INTEGRATION_FLAGS) && $(TASK_DONE); fi
```

Comment:
I don't think integration_flags is defined

Reply:
Nope, it doesn't need to be. It's so that the Makefile that extends this can add in additional flags if it wants to

Reply:
Proclaim doesn't actually have any integration tests